src/encoded/static/components/genome_browser.js

```diff
@@ -879,7 +879,7 @@ class GenomeBrowser extends React.Component {
             // Here we do some approximate math to try to figure out how many lines the labels extend to assuming that ~30 characters fit on one line
             // Labels on the experiment pages are short enough to fit on one line (they contain less information) so we can bypass these calculations for those pages
             if (label === 'cart') {
-                labelLength += file.target ? file.target.label.length + 2 : 0;
+                labelLength += file.target && file.target.label && file.target.label ? file.target.label.length + 2 : 0;
```

Comment:
Repeated `file.target.label`?

Reply:
@forresttanaka I am not sure if I satisfied your comment "just deal with incoming files in the data prop". It is tricky. In series, the association graph is drawn with series files. But non-series files are used in other types. But in series files I also need non-series files to satisfy a requirement to have accession ids draw on the nodes.  If I combine it with the data I get into an issue we both wrestled with where you cannot alter props (to remove non-series files) or else react explodes. That is why I hack-ly added nonseries argument that is only used by series files in assembleGraph().